config/routes.rb

```diff
@@ -1,5 +1,18 @@
 Rails.application.routes.draw do
-  root 'pages#hello'
+  devise_for :users
+  
+  devise_scope :user do
+    authenticated :user do
+      root :to => 'categories#index', as: :authenticated_root
+    end
+    unauthenticated :user do
+      root :to => 'splash#new', as: :unauthenticated_root
+    end
+  end
+
+  resources :categories, only: [:index, :show, :new, :create, :destroy] do
+    resources :payments, only: [:index, :show, :new, :create, :destroy]
```

Comment:
- [ ] Hey @zhorabay, kudos on your impressive work so far! 🌟 I've observed that there are some routes, like show and destroy, in your project that don't seem to be utilized. 🤔 Could you please double-check your routes and ensure that you only include the ones actively used in your project? This will help keep things clean and efficient. Thanks a bunch! 🚀